src/nvim_bridge.rs

```diff
@@ -219,6 +219,12 @@ pub struct CompletionItem {
     pub info: String,
 }
 
+#[derive(Debug)]
+pub struct MessageEntry {
+    pub kind: String,
+    pub content: Vec<(u64, String)>,
```

Comment:
This shows up often enough to be a good candidate for a struct, e.g. `HlContent`

Reply:
`type HlContent = Vec<u64, String>` or `type HlContent = Vec<HlText>; type HlText = (u64, String>` would be better than a struct with a single property.